modules/roles_profiles/manifests/profiles/mac_v3_signing.pp

```diff
@@ -32,15 +32,42 @@
             # we can add worker setup here like in gecko_t_osx_1014_generic_worker.pp
 
             include dirs::tools
+            class { 'scriptworker_prereqs': }
 
-            contain packages::python3_s3
-            file { '/tools/python3':
-                    ensure  => 'link',
-                    target  => '/usr/local/bin/python3',
-                    require => Class['packages::python3_s3'],
-            }
-            contain packages::virtualenv_python3_s3
+            case $::hostname {
```

Comment:
There's no central nodes.pp file, one gets generated by the bootstrapping, so we shouldn't have one committed. So that means moving things into the roles.

Reply:
Ok. I still think a settings.pp would be preferable, though we may have to keep the hostname regexes in the roles.